.github/workflows/android.yml

```diff
@@ -230,9 +233,140 @@ jobs:
           path: ${{ env.SCCACHE_ERROR_LOG }}
           if-no-files-found: 'ignore'
 
+      - name: Prepare build and test artifacts (split)
+        run: |
+          # Prepare directories
+          mkdir -p ${INSTALL_TEST_DIR}/bin/intel64/Release
+          mkdir -p ${INSTALL_DIR}/lib
+
+          # Install tests into tests dir
+          cmake --install ${BUILD_DIR} --config ${{ env.BUILD_TYPE }} --prefix ${INSTALL_TEST_DIR} --component tests
+
+          # Install runtime into install dir; fallback to full install if component is unavailable
+          cmake --install ${BUILD_DIR} --config ${{ env.BUILD_TYPE }} --prefix ${INSTALL_DIR} --component core \
+            || cmake --install ${BUILD_DIR} --config ${{ env.BUILD_TYPE }} --prefix ${INSTALL_DIR}
+
+          # If tests were installed under tests/, move CPU test binaries to expected path
+          if [ -d "${INSTALL_TEST_DIR}/tests" ]; then
+            echo "Moving tests from ${INSTALL_TEST_DIR}/tests/ to ${INSTALL_TEST_DIR}/bin/intel64/Release/"
+            find ${INSTALL_TEST_DIR}/tests -name "*cpu*test*" -type f -exec cp {} ${INSTALL_TEST_DIR}/bin/intel64/Release/ \;
+          fi
+
+          # Add TBB and libc++ shared to runtime package
+          find ${TBB_INSTALL}/lib -name "*.so*" -type f -exec cp {} ${INSTALL_DIR}/lib/ \; 2>/dev/null || echo "No .so files in TBB_INSTALL/lib"
+          cp "${ANDROID_NDK_HOME}/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/lib/${{ env.ANDROID_ABI_CONFIG }}-linux-android/libc++_shared.so" ${INSTALL_DIR}/lib/ || echo "libc++_shared.so not found"
+      - name: Upload CPU test binaries
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
+        with:
+          name: ov_cpu_tests_android_${{ matrix.arch }}
+          path: ${{ env.INSTALL_TEST_DIR }}/
+          if-no-files-found: 'error'
+
+      - name: Upload OpenVINO runtime package (Android)
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
+        with:
+          name: ov_android_package_${{ matrix.arch }}
+          path: ${{ env.INSTALL_DIR }}/
+          if-no-files-found: 'error'
+
+  Test:
```

Comment:
I'd give more specific name to the job (e.g. Unit Tests), in the future we might need to add more test jobs, and this name will be confusing

Reply:
Done